### DISCUSSION TITLE
The whole history is shown in just one line

### DISCUSSION THREAD

**Discussion by rizzini** [PatrickF1/fzf.fish#150] [Q&A] [answered]
Hey, Patrick.

I"m having trouble using your script. It works wonderfully if I install it and doesn't close my terminal. If I close and open it agai, the whole history is shown in just one line

I'm calling it with CTRL + R.
Here is a video I made to show you this -> https://www.youtube.com/watch?v=0zrQXECIFEo 
**Ignore the bottom terminal.**

Edit:
I don't even need to uninstall it. Just install it does the trick.

Edit: 
I tried to add the line below to my config.fish, in order to workaround this problem until I don't fix it, but, interesting enough, the whole history is still shown in one line. If I manually run it, I can search using CTRL + R no problem.
```
if status --is-interactive
   fisher install PatrickF1/fzf.fish
end
```

**Comment by rizzini** [comment, marked as answer]
You can close this thread. I deleted all my Fish config files and it's working now. When I installed Fish a while ago, I played with it with a lot, which lead me with broken config files.

 Sorry for the inconvenience.

**Reply by PatrickF1** [reply]
Glad you figured it out!
